packages/charts/react-charting/src/components/VerticalBarChart/VerticalBarChart.base.tsx

```diff
@@ -700,18 +707,23 @@ export class VerticalBarChartBase extends React.Component<IVerticalBarChartProps
         legendColor: this.state.color,
         shouldHighlight,
       });
-      const barHeight: number = Math.max(yBarScale(point.y), 0);
-      let adjustedBarHeight = 0;
-
-      if (barHeight <= 0) {
+      const barHeight: number = this.props.supportNegativeYValues
+        ? yBarScale(point.y) - yBarScale(0)
+        : yBarScale(point.y);
+      // Calculate threshold for minimum visible bar height
+      const minBarHeight = Math.ceil(yBarScale(this._yMax - this._yMin) / 100.0);
+      let adjustedBarHeight = barHeight;
+
+      if (Math.abs(barHeight) === 0 || (barHeight < 0 && !this.props.supportNegativeYValues)) {
```

Comment:
barHeight <= 0 should be a sufficient condition.

Reply:
using barHeight<=0 will also lead to negative height bars to not render. But now we don't want that. We will now not render anything for barHeight =0 or if (negative value prop is false and bar height is negative).